website/addons/evernote/requirements.txt

```diff
@@ -0,0 +1,2 @@
+evernote==1.25.1
+git+https://github.com/rdhyee/ENML_PY.git@0.0.1
```

Comment:
You shouldn't need this library -- the MFR will be doing this work.

Also, while I know that [ENML prohibits many "dangerous" tags](https://dev.evernote.com/doc/articles/enml.php#prohibited) by design, this still seems like an attack vector, and I'd prefer not using this library in the MFR either. @felliott might have opinions about this too.